DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1558,6 +1568,18 @@ protected override void BanPlayer(int playerIndex)
             }
         }
 
+        private void GhostBanIgnoredUser(string playerName)
+        {
+            var user = connectionManager.UserList.Find(u => u.Name == playerName);
```

Comment:
considering converting to a `Dictionary<string, (string ident, string host, string userName, string extraInfo)>`

Reply:
leave as a TODO in https://github.com/CnCNet/xna-cncnet-client/issues/655